substrate-processor/src/interfaces/substrate.ts

```diff
@@ -53,6 +53,11 @@ export interface SubstrateBlock {
      */
     lastRuntimeUpgrade: unknown
 
+    /**
+     * Hex string representing the block validator
+     */
+    validatorId?: string
```

Comment:
May be call it `validator`?

Reply:
but yeah... just "validator" will be better